README.md

```diff
@@ -47,11 +47,13 @@ Nothing to do :)
 Only for [Single File Components](https://vuejs.org/v2/guide/single-file-components.html) with `<docs>` tag used for documentation.
 
 ```js
-module.exports = storybookBaseConfig => {
-  storybookBaseConfig.module.rules.push({
+module.exports = async ({ config }) => {
+  config.module.rules.push({
```

Comment:
According to the docs, I see that module should export object

```
module.exports = {
  webpackFinal: (config) => { ... }
}
```

https://storybook.js.org/docs/configurations/custom-webpack-config/#full-control-mode


Are you sure, it will work if export functions?

Reply:
Sorry for the super late response, all emails were at the spam folder :(

Reply:
Hey there :) 
Since in the readme a method is exported, I thought that this file would be the `.storybook/webpack.config.js` which works with the code from my PR.
If the example is showing an "inline webpack config" then the setup is different.